source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -112,6 +119,8 @@ export default class NetworkStatusStore extends Store {
     // Passively receive state changes of the cardano-node
     cardanoStateChangeChannel.onReceive(this._handleCardanoNodeStateChange);
 
+    this._getSystemStartTime();
```

Comment:
@daniloprates perhaps we can have this call only for development network? And please be careful - I'm not talking about `globals.environment.isDev` - that one compares NODE_ENV and we need `NETWORK === DEVELOPMENT` check. So please add the `const isDevelopment = NETWORK === DEVELOPMENT;` to `/source/main/environment.js` https://github.com/input-output-hk/daedalus/blob/develop/source/main/environment.js#L30

Reply:
should I move `systemStartTime to `BlockConsolidationStatusPage` as well?

Reply:
No, I think it is fine to leave it in the NetworkStatusStore...